### PR TITLE
Update admin button contents

### LINKED ISSUE #3032: Fix "Run PI as admin to get more data" button cut off on 4k monitors
### Dev Home version

0.503

### Windows build number

10.0.22631.3672

### Other software

_No response_

### Steps to reproduce the bug

DPI issue?

### Expected result

_No response_

### Actual result

button is cut off

### Included System Information

_No response_

### Included Extensions Information

_No response_

### PR DESCRIPTION
**Pull request by zadesai**
## Summary of the pull request
Problem: Run as admin button in multiple pages is cut off
image

Fix: Update the button to have a tooltip similar to the one used in WER page:
image

## PR checklist
- [ ] Closes #3032

### CODE REVIEW

**Comment by dabhattimsft** [comment]
nit: maybe create a common button similar to [CloseButton](https://github.com/microsoft/devhome/blob/main/common/Views/CloseButton.xaml) ?

**Review by dabhattimsft — APPROVED** [review]

File: tools/PI/DevHome.PI/Strings/en-us/Resources.resw [hunk 1]
```diff
@@ -1061,13 +1057,13 @@
...
+    <data name="RunElevatedButton.Text" xml:space="preserve">
     <value>Change</value>
-    <comment>The contents of a button to allow the user to trigger elevation to change WER settings</comment>
+    <comment>The contents of a button to allow the user to trigger elevation</comment>
   </data>
-  <data name="RunElevatedButtonForWER.ToolTipService.ToolTip" xml:space="preserve">
-    <value>Project Ironsides needs to run as Admin in order to modify WER collection for an app.</value>
-    <comment>{Locked="Ironsides"} Tooltip for a button that relaunches Project Ironsides as admin to change WER settings</comment>
+    <data name="RunElevatedButton.ToolTipService.ToolTip" xml:space="preserve">
+    <value>Project Ironsides needs to run as Admin in order to get more data for an app.</value>
```
Comment (timkur) [inline tools/PI/DevHome.PI/Strings/en-us/Resources.resw:1065]: Feels like maybe you need multiple strings here?

In this case, we lost the WER-specific tooltip language (get more data for an app is not why we'd want to elevate for WER).

Also, having the button text say "Change" for the Modules page seems weird, as you're not changing anything.

**Comment by zadesai** [comment]
> nit: maybe create a common button similar to [CloseButton](https://github.com/microsoft/devhome/blob/main/common/Views/CloseButton.xaml) ?

This is a good idea. I tried it, but since we need to pass the page to open and change visibility based on a particular page and helpers associated, it might be ok to have the duplication. Only the button content and tooltip are common, if there are more common parameters in the future, we can create a common class. I will keep an eye on it.

File: tools/PI/DevHome.PI/Strings/en-us/Resources.resw [hunk 2]
```diff
@@ -1069,6 +1065,14 @@
     <value>Project Ironsides needs to run as Admin in order to modify WER collection for an app.</value>
     <comment>{Locked="Ironsides"} Tooltip for a button that relaunches Project Ironsides as admin to change WER settings</comment>
   </data>
+  <data name="RunElevatedButton.Text" xml:space="preserve">
+    <value>Run as admin</value>
```
Comment (timkur) [inline tools/PI/DevHome.PI/Strings/en-us/Resources.resw:1069]: The Icon already means "Run as admin". Want to go for something like "Get Information" to be a bit more descriptive?

**Review by timkur — APPROVED** [review]

**Review by AmelBawa-msft — APPROVED** [review]

**Pull request merged**